modules/search/pages/create-search-indexes.adoc

```diff
@@ -25,28 +25,50 @@ You can export the Search index definition JSON payload from the UI to xref:impo
 
 Couchbase {page-product-name} supports two editing modes for the Search Service: 
 
-* xref:create-quick-index.adoc[Quick Mode]
-* xref:create-search-index-ui.adoc[Advanced Mode]
+* <<quick-mode,Quick Mode>>
+* <<advanced-mode,Advanced Mode>>
+
+[#quick-mode]
+=== Quick Mode Editing
+
+Quick Mode editing is on by default when you xref:create-search-index-ui.adoc[create a new Search index].
+It supports: 
+
+* xref:create-search-index-ui.adoc#configure-settings[Configuring global index settings], which include:
+** xref:customize-index.adoc#analyzers[Setting a default analyzer]
+** xref:customize-index.adoc#date-time[Setting a default date/time parser]
+* Creating xref:customize-index.adoc#type-mappings[Type mappings and mappings]
+* Configuring xref:customize-index.adoc#replica[Replica and partition settings]
+
+[#advanced-mode]
+=== Advanced Mode Editing 
+
+If you select *Enable Advanced Options* to enable Advanced Mode in the Search index editor, the following additional options become available: 
+
+* Creating custom xref:customize-index.adoc#analyzers[analyzers], which can include: 
+** Custom xref:customize-index.adoc#character-filters[character filters]
+** Custom xref:customize-index.adoc#tokenizers[tokenizers]
+** Custom xref:customize-index.adoc#token-filters[token filters], which can use custom xref:customize-index.adoc#wordlists[word lists]
+* Creating custom xref:customize-index.adoc#date-time[date/time parsers]
+* Creating xref:customize-index.adoc#type-mappings[mappings] for Extended Attributes (XATTRs) data in documents
+* Creating xref:customize-index.adoc#type-mappings[mappings] for objects and fields that do not yet exist in your document schema
+* Configuring a xref:customize-index.adoc#type-identifiers[document filter]
+
+All Quick Mode options remain available in Advanced Mode editing. 
 
 [NOTE]
 ====
 For indexes created with Couchbase Server version 7.6 and later, index names must be unique inside a bucket and scope.
-You cannot have 2 indexes with the same name inside the same bucket and scope on a {page-product-name} cluster running version 7.6.
+You cannot have 2 indexes with the same name inside the same bucket and scope on a {page-product-name} operational cluster running version 7.6.
```

Comment:
Little out of scope, but is this just 7.6 or 7.6 and later?

Reply:
Easy enough to fix while we're in here.